webui/react/src/pages/ExperimentList.tsx

```diff
@@ -353,53 +410,68 @@ const ExperimentList: React.FC = () => {
 
     const newColumns = [ ...defaultColumns ].map(column => {
       column.sortOrder = null;
+
       if (column.key === sorter.key) column.sortOrder = sorter.descend ? 'descend' : 'ascend';
-      if (column.key === V1GetExperimentsRequestSortBy.DESCRIPTION) column.render = nameRenderer;
+      if (column.key === 'labels') {
```

Comment:
have you considered moving all the definitions of the columns here instead of importing from an external file and mapping on it to apply tweaks?

Reply:
lol, yeah at this point, there's so much modifications, the external import doesn't make sense.